Blockzilla/UIConstants.swift

```diff
@@ -301,7 +302,7 @@ struct UIConstants {
         static let copyURLToast = NSLocalizedString("browser.copyAddressToast", value: "URL Copied To Clipboard", comment: "Toast displayed after a URL has been copied to the clipboard")
         static let copyMenuButton = NSLocalizedString("Browser.copyMenuLabel", value: "Copy", comment: "Copy URL button in URL long press menu")
         static let eraseButton = NSLocalizedString("URL.eraseButtonLabel", value: "ERASE", comment: "Erase button in the URL bar")
-        static let eraseMessage = NSLocalizedString("URL.eraseMessageLabel", value: "Your browsing history has been erased.", comment: "Message shown after pressing the Erase button")
+        static let eraseMessage = NSLocalizedString("URL.eraseMessageLabel", value: "Browsing history cleared", comment: "Message shown after pressing the Erase button")
```

Comment:
@boek is this change allowed? Or do we need to leave the `URL.eraseMessageLabel` as is, and then introduce a `URL.eraseMessageLabel2`? I remember us doing that on Android to make sure string changes are picked up - not sure if it is needed on iOS?

Reply:
@Delphine Do you know the answer to the above?

Reply:
Confirmed that this needs a new string ID

Reply:
I just updated this in the PR so that we can land it. Thanks @Delphine cc @bogdancernea28

Reply:
@st3fan thank you!